app/javascript/tests/analytics/widgets/MonitoreeFlow.test.js

```diff
@@ -4,19 +4,19 @@ import { Card } from 'react-bootstrap';
 import MonitoreeFlow from '../../../components/analytics/widgets/MonitoreeFlow';
 import mockAnalytics from '../../mocks/mockAnalytics';
 
-const monitoreeFlowTableHeaders = ['Last 24 Hours', 'Last 7 Days', 'Last 14 Days', 'Total'];
-const exposureNewEnrollmentValues = [ 54, 164, 192, 223 ];
-const exposureTransferredInValues = [ 7, 21, 22, 25 ];
-const exposureFromIsolationValues = [ 0, 6, 16, 76 ];
-const exposureClosedValues = [ 4, 11, 12, 15 ];
-const exposureTransferredOutValues = [ 8, 18, 20, 27 ];
-const exposureToIsolationValues = [ 2, 21, 56, 138 ];
-const isolationNewEnrollmentValues = [ 39, 100, 171, 195 ];
-const isolationTransferredInValues = [ 4, 17, 19, 22 ];
-const isolationFromExposureValues = [ 2, 21, 56, 138 ];
-const isolationClosedValues = [ 2, 4, 6, 7 ];
-const isolationTransferredOutValues = [ 5, 18, 22, 25 ];
-const isolationToExposureValues = [ 0, 6, 16, 76 ];
+const monitoreeFlowTableHeaders = [' Last 24 Hours  n (col %) ', ' Last 7 Days  n (col %) ', ' Last 14 Days  n (col %) ', ' Total  n (col %) '];
```

Comment:
so you can actually change the tests below to use `toContain` instead of `toEqual` to prevent any annoying HTML whitespace issues like this!

Reply:
Done https://github.com/SaraAlert/SaraAlert/pull/864/commits/ba7efa72386bde8a121d10cfdf91b6b4a4115381